Package.swift

```diff
@@ -25,9 +25,9 @@ let package = Package(
     dependencies: [
         .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", .exact("2.0.0-beta.3")),
         .package(name: "MapboxGeocoder", url: "https://github.com/mapbox/MapboxGeocoder.swift.git", from: "0.14.0"),
-        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", from: "0.10.8"),
-        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "50.0.0"),
-        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.0.0-beta.20")),
+        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", from: "1.0.0"),
+        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "52.0.0"),
+        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.0.0-beta.21")),
```

Comment:
If we do go ahead and upgrade to v10.0.0-rc.1 in this PR, it’ll require us to also upgrade to Turf v2.0.0-beta.1 and MapboxDirections v2.0.0-beta.4.

Reply:
I think we'll not be able to update just yet because Mapbox Maps [v10.0.0-rc.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.0.0-rc.1) depends on Mapbox Common `v14.0.1`.